File: src/test/java/test/AddDepartmentMembersTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class AddDepartmentMembersTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        AddDepartmentMembersReqDto request = new AddDepartmentMembersReqDto();
        request.setDepartmentId("departmentId_9612");
        request.setOrganizationCode("organizationCode_5264");
        request.setUserIds(Collections.singletonList("String_8146"));

        IsSuccessRespDto response = client.addDepartmentMembers(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/AddDepartmentMembersTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.AddDepartmentMembersReqDto{
        UserIds: nil,
        OrganizationCode: "organizationCode_527",
        DepartmentId: "departmentId_9419",
        DepartmentIdType: ,
    }
    response := client.AddDepartmentMembers(&request)
    fmt.Println(response)
}

